pkg/antctl/raw/multicluster/leave.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package multicluster
+
+import (
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"antrea.io/antrea/pkg/antctl/raw/multicluster/common"
+)
+
+var leaveOpts *common.CleanOptions
+
+var leaveExamples = strings.Trim(`
+# Leave the ClusterSet.
+  antctl mc leave --clusterset test-clusterset
+`, "\n")
+
+func NewLeaveCommand() *cobra.Command {
+	command := &cobra.Command{
+		Use:     "leave",
+		Short:   "Leave ClusterSet from a member cluster",
+		Args:    cobra.MaximumNArgs(0),
+		Example: leaveExamples,
+		RunE:    leaveRunE,
+	}
+
+	o := common.CleanOptions{}
+	leaveOpts = &o
+	command.Flags().StringVarP(&o.Namespace, "namespace", "n", "kube-system", "Antrea Multi-cluster Namespace. Default to kube-system")
```

Comment:
Please define a constant for "kube-system", like defaultMemberNamespace.

Reply:
Defaults to

Reply:
Could we use the Namespace in the current-context of kubeconfig? @luolanzone

Reply:
We can, but for member cluster, it will be installed on 'kube-system' by default. It's the Namespace used in the member manifest. so I feel it's better to use 'kube-system'.

Reply:
Sure. We can go with `kube-system` too. Just define a constant.